lib/dataset/aligned_dict.cpp

```diff
@@ -330,27 +337,56 @@ Dict<Key, Value> Dict<Key, Value>::merge_from(const Dict &other) const {
 }
 
 template <class Key, class Value>
-bool Dict<Key, Value>::item_applies_to(const Key &key,
-                                       const Dimensions &dims) const {
-  const auto &val = m_items.at(key);
+bool AlignedDict<Key, Value>::item_applies_to(const Key &key,
+                                              const Dimensions &dims) const {
+  const auto &val = m_items[key];
```

Comment:
I would prefer keeping the `at`: It makes it clear to the reader that this will *not* result in insertion, even if they don't know that the map being used is a special implementation.

Reply:
Assuming that we keep the C++ naming convention. If we switch to Python names, should we remove `at`?

Reply:
I think we should keep naming from `std` at least for std-like containers?